### ISSUE TITLE
listener_stop bug

### ISSUE THREAD

**Issue by optechadam** [nicocha30/ligolo-ng#56]
After terminating a listener on the agent I get the following repeating error:

```
ERRO[0354] Listener relay failed with error: EOF         listener="#1 [tgt@DESKTOP-AQ78B2] (tcp) [Agent] 0.0.0.0:1234 => [Proxy] 127.0.0.1:4321"
WARN[0354] Listener failed. Restarting in 5 seconds...
```
The agent does not have a listening port anymore and no errors on the agent terminal output.

**Comment by lap012** [comment]
same for me.
![image](https://github.com/nicocha30/ligolo-ng/assets/156910856/39be3722-7e52-4992-8214-01d84852d1bc)

**Comment by helmdael** [comment]
I have the exact same issue when I try to stop the listener using agent and proxy 0.5.x linux amd64 binaries.
When I use 0.4.5 linux_amd64 version the listener stops without issues.

My box is running as a vm (Linux kali 6.6.9-amd64 #1 SMP PREEMPT_DYNAMIC Kali 6.6.9-1kali1 (2024-01-08) x86_64 GNU/Linux)

![image](https://github.com/nicocha30/ligolo-ng/assets/80857002/c02df997-aa74-4f65-bde1-ba1c4e914f98)

**Comment by scr4sh** [comment]
Same Problem!

**Comment by deadloot** [comment]
having the same problem. proxy is flooded with errors on closed connection.

are there ways or plans to suppress errors, kill the restart process, etc that could be useful but also temporarily resolve the error-spam?

**Comment by partyh4t** [comment]
Same issue here! Not only does it spam the terminal, but its also causing any other listeners i have running to become dysfunctional and not work as intended. Have to restart the entire program.

**Comment by jdholtz** [comment]
This should be fixed in #79. Can anyone test just to verify the fix works for them too?
